docs/available_commands.md

```diff
@@ -164,31 +277,37 @@ So only a message that the component was not found and could not be updated is l
 
-### Use of local schema
+You can select the schema with the `--schema-type` and `--schema-path` options:
 
-With the `--schema-path` flag, users can supply their own schema to the validator.
+    cdx-ev validate bom.json [--schema-type default]           # stock CycloneDX schema
+    cdx-ev validate bom.json --schema-type custom              # built-in custom schema
+    cdx-ev validate bom.json --schema-path <json_schema.json>  # your own schema
 
-    cdx-ev validate bom.json --schema-path=C:\users\documents\sbom_schemas\example_schema.json  # uses a schema "example_schema.json" saved on the users computer to verify the sbom
+For all built-in schemas, the tool attempts to determine the correct CycloneDX version from the input SBOM and falls back to version 1.3 if that fails. The following versions are currently supported:
+
+| Type | Supported CycloneDX versions |
+| ---- | ---------------------------- |
+| `default` | 1.2 to 1.5 |
+| `custom` | 1.3 to 1.5 |
 
 ### Validation of filename
 
 The tool, by default, also validates the filename of the SBOM. Which filenames are accepted depends on several command-line options:
 
 * `--no-filename-validation` completely disables validation.
 * Use `--filename-pattern` to provide a custom regex. The filename must be a full match, regex anchors (^ and $) are not required. Regex patterns often include special characters. Pay attention to escaping rules for your shell to ensure proper results.
-* In all other cases, the acceptable filenames depend on the `--schema-type` option:
-  * Using the `default` schema (i.e., vanilla CycloneDX), the validator accepts the two patterns recommended by the [CycloneDX specification](https://cyclonedx.org/specification/overview/#recognized-file-patterns): `bom.json` or `*.cdx.json`.
-  * Using the `custom` schema, filenames must match one of these patterns: `bom.json` or `<name>_<version>_<hash>|<timestamp>|<hash>_<timestamp>.cdx.json`. Read on for some clarifications.
+* In all other cases, the acceptable filenames depend on the selected schema:
+  * When using the stock CycloneDX schema (`--schema-type default` or no option at all) or when using your own schema (`--schema-path` option), the validator accepts the two patterns recommended by the [CycloneDX specification](https://cyclonedx.org/specification/overview/#recognized-file-patterns): `bom.json` or `*.cdx.json`.
+  * When validating against the built-in custom schema (`--schema-type custom`), filenames must match one of these patterns: `bom.json` or `<name>_<version>_<hash>|<timestamp>|<hash>_<timestamp>.cdx.json`. Read on for some clarifications.
 
 `<name>` and `<version>` correspond to the respective fields in `metadata.component` in the SBOM.
```

Comment:
is this really correctly formatted? Seems odd that the information for the last bullet point is two indentations less.

Reply:
Sorry, I don't follow. Lines 308 and 309 have the same intendation, as I intended.

Or do you refer to the fact that lines 312 and 314 are not part of the list. That's also as intended but I agree it's not pretty. I just wanted to avoid another level of intendation.

Reply:
I was talking about line 312. The sentence "`<name>` and `<version>` correspond to the respective fields in `metadata.component` in the SBOM." This sentence refers to the last bullet point in line 310, right? But you already answered it 

> That's also as intended but I agree it's not pretty. I just wanted to avoid another level of intendation.